### PR TITLE
systemd: Make Shut Down action a proper verb instead of a noun

### PR DESCRIPTION
**Pull request by andreasn**
Shutdown is not a verb, it is a state, Shut Down is an action

Fixes issue #5414

### CODE REVIEW

**Comment by stefwalter** [comment]
Andreas, it looks like some of the tests also need fixing. In particular the file ```test/verify/check-shutdown-restart``` needs fixing on lines like this:

```
    b.click("#shutdown-group a:contains('Shutdown')")
```

File: pkg/systemd/host.js [hunk 1]
```diff
@@ -1304,9 +1304,9 @@ PageShutdownDialog.prototype = {
         self.delay = $("#shutdown-delay li:first-child").attr("value");
 
         if (PageShutdownDialog.type == 'shutdown') {
-          $('#shutdown-dialog .modal-title').text(_("Shutdown"));
+          $('#shutdown-dialog .modal-title').text(_("Shut Down"));
           $("#shutdown-action").click($.proxy(this, "shutdown"));
-          $("#shutdown-action").text(_("Shutdown"));
+          $("#shutdown-action").text(_("Shut Down"));
```
Comment (dperpeet) [inline pkg/systemd/host.js]: Is there a reason this isn't `Shut down`?

Comment (andreasn) [inline pkg/systemd/host.js]: Headline case for buttons
See Capitalization for Common Components here: http://www.patternfly.org/styles/terminology-and-wording/#_

**Comment by andreasn** [comment]
> Andreas, it looks like some of the tests also need fixing. 

Fixed.

A lot of the po-files mention `msgid "Shutdown"` Does this need to be modified as well?

**Comment by andreasn** [comment]
> Does this need to be modified as well?

Seems it does. Will fix

**Comment by andreasn** [comment]
Seems the translations should not be changed manually, so removed those changes from the commit again.

**Pull request closed without merging**